### ISSUE TITLE
java.util.MissingFormatArgumentException: Format specifier '%s'

### ISSUE THREAD

**Issue by jgrzebyta** [degree9/boot-semver#65]
Affected version **1.6.0**

Full debug log:
```
Initial Project Version...: 0.2.0
Generating Version Namespace...: rdf4j.version
Current Build Version...: 0.2.1-SNAPSHOT
                              java.lang.Thread.run              Thread.java:  748
java.util.concurrent.ThreadPoolExecutor$Worker.run  ThreadPoolExecutor.java:  617
 java.util.concurrent.ThreadPoolExecutor.runWorker  ThreadPoolExecutor.java: 1142
               java.util.concurrent.FutureTask.run          FutureTask.java:  266
                                               ...                               
               clojure.core/binding-conveyor-fn/fn                 core.clj: 1938
                                 boot.core/boot/fn                 core.clj: 1029
                               boot.core/run-tasks                 core.clj: 1019
       degree9.boot-semver.impl/version-impl/fn/fn                 impl.clj:  100
        degree9.boot-semver.impl/eval1026/fn/fn/fn                 impl.clj:   93
                                               ...                               
                                    boot.util/info                 util.clj:   84
                                  boot.util/print*                 util.clj:   57
                                clojure.core/apply                 core.clj:  646
                                               ...                               
                               clojure.core/format                 core.clj: 5527
                               clojure.core/format                 core.clj: 5533
                           java.lang.String.format              String.java: 2940
                        java.util.Formatter.format           Formatter.java: 2455
                        java.util.Formatter.format           Formatter.java: 2519
java.util.MissingFormatArgumentException: Format specifier '%s'
    formatSpecifier: "%s"
              clojure.lang.ExceptionInfo: Format specifier '%s'
    file: "/tmp/boot.user4367583297836020968.clj"
    line: 37
```

In line 37 of the boot script I have:
```
(clojure.core/let [boot?__1744__auto__ true] (clojure.core/if-not boot?__1744__auto__ (clojure.core/when-let [main__1745__auto__ (clojure.core/resolve (quote boot.user/-main))] (main__1745__auto__ "develop" "build-standalone")) (boot.core/boot "develop" "build-standalone")))

```

**Comment by burn2delete** [comment]
Please try version `1.7.0-SNAPSHOT`

**Comment by jgrzebyta** [comment]
Fixed in `1.7.0-SNAPSHOT`
